### PR TITLE
repository content added

### PR DESCRIPTION
**Pull request by wesleymccollam**
- CHANGELOG.md
- CONTRIBUTING.md
- LICENSE.md
- README.md (dev steps, community info, repo structure)

### CODE REVIEW

File: contributing/development.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+# Development Environment Setup
+## Prerequisites
+The following applications must be installed locally to run the provider:
+- [Terraform](https://www.terraform.io/downloads.html) 1.0+ (to run acceptance tests)
+- [Go](https://golang.org/doc/install) 1.20+ (to build the provider plugin)
+
+To run the example in this repository, you will also need
+- [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/)
```
Comment (henryrecker-pingidentity) [inline contributing/development.md:8]: I believe docker compose is included with docker now so we could remove the second part here if we want

File: docker-compose/docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+#-   |                       |    username: administrator                             |
+#-   |                       |    password: 2FederateM0re                             |
+#-   +-----------------------+--------------------------------------------------------+
+#-   |  PingFederate         |  https://localhost:9999/pingfederate/app               |
+#-   +-----------------------+--------------------------------------------------------+
+#-------------------------------------------------------------------------------------------------------------------------------
+
+services:
+  pingfederate:
+    image: pingidentity/pingfederate:2305
```
Comment (henryrecker-pingidentity) [inline docker-compose/docker-compose.yaml:23]: I'd use a tag with the actual PingFed version here for clarity

**Review by henryrecker-pingidentity — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# PingFederate Terraform Provider
+The PingFederate Terraform provider is a plugin for [Terraform](https://www.terraform.io/) that supports the management of PingFederate configuration. This provider is maintained internally by the Ping Identity team.
+
+# Disclaimer - Provider in Development
+The PingFederate Terraform provider is still in development, and breaking changes are likely. The MVP for this provider is set to release at the end of Q3 in 2023. To see the current resources, navigagate to the [Changelog](CHANGELOG.md). The resource types added to this repository are considered complete, however will undergo additional testing before this provider is published to the Terraform Registry.
```
[the remark this answers is not in the document]
Comment (wesleymccollam) [reply README.md]: fixed typo

File: README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+## Requirements
+* Terraform 1.1+
+* Go 1.20+
```
[the remark this answers is not in the document]
Comment (wesleymccollam) [reply README.md]: added "(for local development builds)"

File: contributing/development.md (hunk 1 above)
Comment (wesleymccollam) [reply contributing/development.md]: removed the second part of the line containing docker-compose info

File: docker-compose/docker-compose.yaml (hunk 2 above)
Comment (wesleymccollam) [reply docker-compose/docker-compose.yaml]: 2305-11.2.5 added

**Pull request merged**